doc/docportal/other_platforms/android.rst

```diff
@@ -59,6 +59,59 @@ There are a few other options to transfer the game files:
 See :doc:`../use_scummvm/game_files` for more information about game file requirements.
 
 
+Adding SAF paths to ScummVM directory list
+==========================================
+In this new version of ScummVM Android, significant changes were made to the file access system to allow support for modern versions of the Android Operating System.
+If you find that your existing added games or custom paths no longer work, please edit those paths and this time use the SAF system to browse to the desired locations.
+To do that:
+
+  1. For each game whose data is not found, go to the "Paths" tab in the "Game Options" and change the "Game path"
+
+  2. Inside the ScummVM file browser, use "Go Up" until you reach the "root" folder where you will see the "<Add a new folder>" option.
+
+.. figure:: ../images/android/browser-root.png
+
+    File Browser root with <Add a new folder> item
+
+  3. Choose that, then browse and select the "parent" folder for your games subfolders, e.g. "SD Card > myGames". Click on "Use this folder".
+
+.. figure:: ../images/android/fs-root.png
+
+    OS file browser root
+
+.. figure:: ../images/android/fs-folder.png
+
+    OS file browser selectable folder with "Use this folder" button
+
+.. figure:: ../images/android/fs-permission.png
+
+    OS file browser ask to grant ScummVM directory access permission
+
+  4. Then, a new folder "myGames" will appear on the "root" folder of the ScummVM browser.
```

Comment:
As above, change "myGames" to "ScummVMgames" to be consistent with what is shown in the screenshot.

Reply:
Done